### ISSUE TITLE
[10.2,10.5][Accessibility]Pictogrammes

### ISSUE THREAD

**Issue by jfrocha** [informed-governance-project/NISINP#378]
Lorsque les CSS ne sont pas chargés, les pictogrammes présents dans la colonne `« Statut »` ne sont plus présents. 

Pour y remédier, intégrer une balise `<span class="visually-hidden">` au-dessus ou en dessous des balises `<i>` de ces éléments et la renseigner en indiquant sa destination.

Pour ne pas faire doubles informations, ajouter un attribut `aria-hidden= "true"` dans ces mêmes balises `<i> `et idéalement, supprimer les attributs aria-label présents. 

Pour obtenir par exemple concrètement :

```
<i class="logo-report-ongoing logo-status" aria-hidden= "true"></i>
<span class="visually-hidden"> L'incident est toujours en cours</span>
```

En l’absence de définition de couleurs de fond, certaines personnes ne peuvent pas personnaliser les couleurs via leur technologie d’assistance.

•	. logo-report-impact
•	. logo-report-ongoing
•	. logo-report-closed
•	. logo-report-unsubmitted
•	. logo-report-under-review
•	. logo-report-overdue
•	. logo-report-pass
•	. logo-report-fail

Pour y remédier, dans les CSS, pour les sélecteurs ci-dessous, ajouter une déclaration de couleur de fond `background-color: #XXX;.`
